### PR TITLE
(GROWUX-1361) busd

### PR DESCRIPTION
**Pull request by akuter-bc**

### CODE REVIEW

**Review by jjambazov-bc — APPROVED** [review]

File: chain/polygon/tokens.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "address": "0xdab529f40e671a1d4bf91361c21bf9f0c9712ab7",
+        "decimals": 18,
+        "displaySymbol": "BUSD",
+        "logo": "",
```
Comment (pavel-bc) [inline chain/polygon/tokens.json:6]: @akuter-bc Shouldn't this one have a URL?

Comment (akuter-bc) [reply chain/polygon/tokens.json]: yes! thanks

**Comment by jhuntbach-bc** [comment]
Do we want to use https://polygonscan.com/token/0x9c9e5fd8bbc25984b178fdce6117defa39d2db39 or https://polygonscan.com/token/0xdab529f40e671a1d4bf91361c21bf9f0c9712ab7 ?

File: extensions/blockchains/polygon/assets/0x9c9e5fd8bbc25984b178fdce6117defa39d2db39/info.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "binance-usd",
```
Comment (jhuntbach-bc) [inline extensions/blockchains/polygon/assets/0x9c9e5fd8bbc25984b178fdce6117defa39d2db39/info.json:2]: Should the name be `Binance USD`?

Comment (akuter-bc) [reply extensions/blockchains/polygon/assets/0x9c9e5fd8bbc25984b178fdce6117defa39d2db39/info.json:2]: spoke with seba and decided we want to use what the contract says. so this should actually be BUSD Token.
What's your thoughts?

**Comment by pavel-bc** [comment]
@jhuntbach-bc Good catch! Which one is listed on the official website? Why is there a mismatch - was there any migration?

**Review by pavel-bc — APPROVED** [review]
Proceeding with official token - https://www.binance.com/en/blog/markets/binancepeg-busd-is-now-available-on-avalanche-and-polygon-849635478180235964

**Pull request merged**